template-deno-lit/vite.config.mjs

```diff
@@ -1,6 +1,6 @@
-import { defineConfig } from 'npm:vite'
+import { defineConfig } from 'npm:vite@^3.1.3'
```

Comment:
Is there a reason it's not using `3.2.3` here as it's the current latest version?

Reply:
I copied the versions used in the package.jsons in the repo (ex. https://github.com/bluwy/create-vite-extra/blob/master/template-ssr-vue/package.json)

Reply:
Ah alright. I can tackle the version bump separately then. Thanks!